### PR TITLE
Added tooltip and change the size of description

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
Added tooltip and change the size of description

### CODE REVIEW

File: widget/edit.html [hunk 1]
```diff
@@ -77,9 +83,11 @@ <h6>Converts a given JSON input to an actionable Grid visualization</h6>
...
-                        <ul id="action-editor-without-record" class="list-group margin-top-md clearfix margin-bottom-0" data-as-sortable
-                            data-ng-model="config.selectedPlaybooksWithoutRecord" data-ng-show="playbookData">
-                            <li id="without-record-draggable-item-all{{$index}}" class="column-editor-block margin-bottom-sm"
+                        <ul id="action-editor-without-record" class="list-group margin-top-md clearfix margin-bottom-0"
+                            data-as-sortable data-ng-model="config.selectedPlaybooksWithoutRecord"
+                            data-ng-show="playbookData">
+                            <li id="without-record-draggable-item-all{{$index}}"
+                                class="column-editor-block margin-bottom-sm"
                                 data-ng-repeat="action in config.selectedPlaybooksWithoutRecord" data-as-sortable-item>
                                 <div cs-icons data-ng-model="action.icon" style="width: 50px; display: inline-block;">
```
Comment (cs-shabaz-salmani) [inline widget/edit.html]: Avoid using styles, please use predefined classes for same
e.g width-50 and display-inline-block

File: widget/edit.html [hunk 2]
```diff
@@ -107,9 +115,11 @@ <h6>Converts a given JSON input to an actionable Grid visualization</h6>
...
-                        <ul id="action-editor-with-record" class="list-group margin-top-md clearfix margin-bottom-0" data-as-sortable
-                            data-ng-model="config.selectedPlaybooksWithRecord" data-ng-show="playbookData">
-                            <li id="with-record-draggable-item-all{{$index}}" class="column-editor-block margin-bottom-sm"
+                        <ul id="action-editor-with-record" class="list-group margin-top-md clearfix margin-bottom-0"
+                            data-as-sortable data-ng-model="config.selectedPlaybooksWithRecord"
+                            data-ng-show="playbookData">
+                            <li id="with-record-draggable-item-all{{$index}}"
+                                class="column-editor-block margin-bottom-sm"
                                 data-ng-repeat="action in config.selectedPlaybooksWithRecord" data-as-sortable-item>
                                 <div cs-icons data-ng-model="action.icon" style="width: 50px; display: inline-block;">
```
Comment (cs-shabaz-salmani) [inline widget/edit.html]: please use predefined classes

File: widget/edit.html (hunk 1 above)
Comment (cs-sujeet-ahirrao) [reply widget/edit.html]: Done

File: widget/edit.html (hunk 2 above)
Comment (cs-sujeet-ahirrao) [reply widget/edit.html]: done

**Review by cs-pratap-nikam — APPROVED** [review]
Looks Good

**Pull request merged**